Anagram.java

```diff
@@ -28,22 +30,65 @@ public static void main(String args[]) {
 
 	// Returns true if the two given strings are anagrams, false otherwise.
 	public static boolean isAnagram(String str1, String str2) {
-		// Replace the following statement with your code
-		return false;
+		String str1Check = preProcess(str1);
+		String str2Check = preProcess(str2);
+		int checker = 0;
+		for (int i = 0; i < str2Check.length() ; i++){
+			for (int j = 0; j < str1Check.length() ; j++){
+				if (str2Check.charAt(i) == str1Check.charAt(j)){
+					checker++;
+				}
+			}
+			if (checker == 0){
+				return  false;
+			}
+			checker = 0;
+		}
+		return true;
```

Comment:
This is incorrect code, let me give few examples

1. str1Check = "aa"; str2Check = "a";
Since lengths are not equal // **expected -> false**

where i = 0; checker after second loop is 2. not enter if.. exit loop 
**actual: true**

2. str1Check = "aab"; str2Check = "abb"; number of appears of a and b is different
Therefore expected: "false"

Your loops checker values per i:
1. i = 0 -> checker = 2
2. i = 1 -> checker = 1
3. i = 2 -> checker = 1

**actual: true**

I am not going to deduct points on this.. 
However, as practice to the midterm exam I want you to build isAnagram function. from scratch 
Sent it to me for review..